packages/Webkul/Installer/src/Resources/lang/bg/app.php

```diff
@@ -0,0 +1,822 @@
+                    'meta-description'  => 'Мета описание 11',
+                    'meta-keywords'     => 'мета1, мета2, мета3',
+                    'meta-title'        => 'Мета заглавие',
+                    'name'              => 'Продукт 11',
+                    'short-description' => 'Известен факт е, че читателя обръща внимание на съдържанието, което чете, а не на оформлението му. Свойството на Lorem Ipsum е, че до голяма степен има нормално разпределение на буквите и се чете по-лесно, за разлика от нормален текст на английски език като "Това е съдържание, това е съдържание".',
+                ],
+            ],
+
+            'product-bundle-option-translations' => [
+                '1' => [
+                    'label' => 'Пакет Вариант 1',
+                ],
+
+                '2' => [
+                    'label' => 'Пакет Вариант 1',
+                ],
+
+                '3' => [
+                    'label' => 'Пакет Вариант 2',
+                ],
+
+                '4' => [
+                    'label' => 'Пакет Вариант 2',
+                ],
+            ],
+        ],
+    ],
+
+    'installer' => [
+        'index' => [
+            'create-administrator' => [
+                'admin'            => 'Администратор',
+                'bagisto'          => 'Bagisto',
+                'confirm-password' => 'Потвърдете парола',
+                'email'            => 'Имейл',
+                'email-address'    => 'admin@example.com',
+                'password'         => 'Парола',
+                'title'            => 'Създай Администратор',
+            ],
+
+            'environment-configuration' => [
+                'algerian-dinar'              => 'Алжирски динар (DZD)',
+                'allowed-currencies'          => 'Разрешени валути',
+                'allowed-locales'             => 'Разрешени езици',
+                'application-name'            => 'Име на приложението',
+                'argentine-peso'              => 'Аржентинско песо (ARS)',
+                'australian-dollar'           => 'Австралийски долар (AUD)',
+                'bagisto'                     => 'Bagisto',
+                'bangladeshi-taka'            => 'Бангладешка така (BDT)',
+                'bahraini-dinar'              => 'Бахрейнски динар (BHD)',
+                'brazilian-real'              => 'Бразилски реал (BRL)',
+                'british-pound-sterling'      => 'Британски лири стерлинги (GBP)',
+                'bulgarian-lev'               => 'Български лев (BGN)',
```

Comment:
The `bulgarian-lev` key should be added in all locales in the installer package.